build/common.ps1

```diff
@@ -122,6 +122,7 @@ Function Invoke-BuildStep {
                 Trace-Log "[STOPPED +$(Format-ElapsedTime $sw.Elapsed)] $BuildStep"
             }
             else {
+                $err
```

Comment:
what this do? this doesn't seem like a known variable, I only found $Error

Reply:
I think it was an attempt at debugging before I figured out the catch block on lines 109-111, which was added in another PR which I actually started working on after I started this PR's branch.